docs/tutorial/part01.rst

```diff
@@ -2,5 +2,83 @@
 Set up the environment.
 =======================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+The following section will have the steps to set your local environment, so you can begin to work on your Django-neomodel implementation.
+If you want a point to start, we provide a base app to begin the tutorial. First clone the paradise-papers-django_ repo.
+
+    .. _paradise-papers-django: https://github.com/neo4j-examples/paradise-papers-django
+
+After you download the repo go to the ``start_app`` branch and you will get a working app that will be our start point.
+
+.. [*] Note:
+    you can use your own app to implement this tutorial and if you download the the start branch you will have only the base django project
+     with the front-end implementation.
+
+Set your virtualenv
+===================
+
+First you need Python and pip install on your local environment. Here is de official download page of python_
+
+    .. _python: https://www.python.org/downloads/
+
+Here is a quick tutorial on how to install pip_
+
+    .. _pip: https://pip.pypa.io/en/stable/installing/#installing-with-get-pip-py
+
+Now we have everything we need to create our virtualenv. On the next part we are going to introduce the basic to get your app running; if you want
+more information check this virtualenv_ tutorial
+
+    .. _virtualenv: https://virtualenv.pypa.io/en/stable/
+
+First go to your app folder on the case of the paradise-papers-django project you need to be on the main folder (localPath/ paradise-papers-django)
+and run the following command to install virtualenv::
+
+    pip install virtualenv
+
+You can use this pip command to check the dependencies installed::
+
+    pip freeze
+
+After you install the virtualenv you need to create a virtual environment for you app. On the same folder that you install virtualenv you need to run the

```

Comment:
after  the virtualenv, I think it would be useful to have a comma or a semi-column.